fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Weight distribution plot\n",
+    "bins=np.arange(np.sqrt(len(np.concatenate(matdropped.values))))\n",
+    "bins = (bins - np.min(bins))/np.ptp(bins)\n",
+    "fig, axes = plt.subplots(1,2, figsize=(15,5))\n",
+    "\n",
+    "# Distribution of raw weights\n",
+    "rawdist = sns.distplot(matdropped.values.flatten(), bins=bins, kde=True, ax=axes[0])\n",
+    "rawdist.set(xlabel='Correlation Values', ylabel = 'Density Frequency')\n",
+    "\n",
+    "# Probability density of log10\n",
```

Comment:
What is the purpose of looking at the log of the weights also?

Reply:
Was something for the JN that I just copied. Got it from Fornito's book. Removed it from this tutorial.